postgres/datadog_checks/postgres/statements.py

```diff
@@ -32,41 +32,36 @@
   LIMIT {limit}
 """
 
+PG_STAT_STATEMENTS_COLUMN_QUERY = 'SELECT * FROM {pg_stat_statements_view}'
```

Comment:
Add a `limit 0` here as to figure out the columns we don't actually need to load any data. 

See the existing comment `# Querying over '*' with limit 0 allows fetching only the column names from the cursor without data`